stock_scheduler_assignation_horizon: wider settings approach (#1131)
OCA/stock-logistics-workflow: stock_scheduler_assignation_horizon/readme/DESCRIPTION.rst

This module allows to set a time horizon that will prevent the scheduler from marking stock
pickings as *ready* before the indicated number of days (horizon).

> Options can be enabled globally and for each company separately, remember that to enable
> mode for the company scheduler needs to be modified to set the company correctly.

Reviewer: ```suggestion

This behavior can be enabled globally via settings or for each company separately.
```
Can you please add a CONFIGURE.rst?
